demo/main.gd

```diff
@@ -1,24 +1,22 @@
 extends Node3D
 
 var xr_interface : XRInterface = null
-var scene_capture: OpenXRFbSceneCaptureExtensionWrapper = null
 
 # Called when the node enters the scene tree for the first time.
 func _ready():
 	xr_interface = XRServer.find_interface("OpenXR")
 	if xr_interface and xr_interface.is_initialized():
 		var vp: Viewport = get_viewport()
 		vp.use_xr = true
-		
-	scene_capture = OpenXRFbSceneCaptureExtensionWrapper.get_singleton()
-	scene_capture.connect("scene_capture_completed", _on_scene_capture_completed)
+
+	OpenXRFbSceneCaptureExtensionWrapper.scene_capture_completed.connect(self._on_scene_capture_completed)
```

Comment:
Let's use `Engine.get_singleton("OpenXRFbSceneCaptureExtensionWrapper")` instead as the current approach causes syntax error until the editor is restarted a second time.

@dsnopek On the syntax error, is that a known issue, or should I be create a GDExtension issue for it?

Reply:
Ok!

It's a known GDExtension issue. But it should only happen the first time you open the Godot editor with that GDExtension - all subsequent times should be fine (a file in the `.godot/` directory gets generated). This will get fixed sooner or later :-)

Reply:
Updated in my latest push